### ISSUE TITLE
Show and edit .txt

### ISSUE THREAD

**Issue by JorisBodin** [Infomaniak/android-kDrive#977]

**Comment by szvan** [comment]
j'ai le même probleme:
je veux ouvrir un fichier txt ou .MD avec markor mais ce dernier me mets a chaque fois qu'il ne peut éditer que des fichiers locaux.. avant les dernières MAJ  cela marchait en activant l'option du fichier "disponible hors ligne". 

<img width="720" height="1600" alt="Image" src="https://github.com/user-attachments/assets/6f3392b6-7d3a-4b6b-a818-c8f67d385825" />

**Comment by Lnamw** [comment]
Bonjour,
Merci pour votre message.
Pourriez-vous essayer d’effectuer l’édition avec une autre application que  Markor et nous confirmer si cela fonctionne ?
De notre côté des modifications ont été apportées et l'édition fonctionnera directement depuis kDrive dans les prochains jours.

**Comment by szvan** [comment]
Effectivement, avec un autre éditeur cela fonctionne. et à ce jour l’implémentation d'editeur à Kdrive fonctionne aussi, parfait
